requirements.txt

```diff
@@ -0,0 +1,9 @@
+allure-pytest>=2.8.40
+allure-python-commons>=2.8.40
+isort>=5.8.0
+helium>=3.0.6
+pur>=5.4.0
```

Comment:
https://pypi.org/project/pur/

Running pur on that file updates the packages to current latest versions:
pur -r requirements.txt